### DISCUSSION TITLE
Connect Multiple Drones

### DISCUSSION THREAD

**Discussion by mmarci22** [bitcraze/discussions#1040] [Q&A]
Hi, 

Is it possible to connect two separate Crazyflie drones to one computer via one radio?

**Comment by knmcguire** [comment]
Yes, it is!

Not through the cfclient but we have a swarming class in our crazyflie python library. There is a step-by-step guide to be found here:

https://www.bitcraze.io/documentation/repository/crazyflie-lib-python/master/user-guides/sbs_swarm_interface/

Also make sure to check out the crazyswarm2 project if you are a bit familiar with ROS2: https://imrclab.github.io/crazyswarm2/
